tests/graphql/transaction.spec.ts

```diff
@@ -204,6 +209,146 @@ describe("Transaction", () => {
     });
   });
 
+  describe("#createSplit", () => {
+    let client: Client;
+    let stub: any;
+
+    before(() => {
+      client = createClient();
+      stub = sinon.stub(client.graphQL, "rawQuery");
+    });
+
+    after(() => {
+      stub.restore();
+    });
+
+    it("should call rawQuery once using proper arguments and return transaction data", async () => {
+      // arrange
+      const splitData = [
+        {
+          id: 1,
+          amount: -500,
+          category: TransactionCategory.Vat_7
+        },
+        {
+          id: 2,
+          amount: -500,
+          category: TransactionCategory.Vat_19
+        },
+      ];
+      const transactionData = createTransaction({
+        amount: -1000,
+        category: TransactionCategory.Private,
```

Comment:
I don't think this is possible, we won't be able to have a transaction with a category as "PRIVATE" and have splits. It doesn't make a difference for the tests, it just might be unnecessary. But anyway, the tests aren't testing that part, so the tests are good!

Reply:
Thanks for the input 🙌! You are right, after a split the parent transaction's `category` will be cleared. I updated the tests in https://github.com/kontist/js-sdk/pull/92/commits/8cdd542e662ca14940bdf50393a80b936e24eb07 to now consist of real-world examples. Also, I added some verbosity which makes the tests a bit lengthy but hopefully more comprehensible.